SPECS/libtool/libtool.spec

```diff
@@ -1,53 +1,62 @@
+%define sha1    libtool=3e7504b832eb2dd23170c91b6af72e15b56eb94e
```

Comment:
Ask: could you remove the `sha1` macro. It's a remnant of old times and we tend to clean these up as we go.